README.md

```diff
@@ -1,27 +1,4 @@
 <a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
```

Comment:
As requested by the previous reviewer, please kindly use correct Github flow in your preject: 
MEANING: You are required to create a pull request from `feature branch` to the `main branch`. 
`feature branch` is a branch you have worked on with the new features.
 
Learn more about [Correct Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)